platform_versioned_docs/version-24.1/data_studios/index.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: "Overview"
+description: "Data Studios public preview."
+date: "1 Oct 2024"
+tags: [data, studios]
+---
+
+Data Studios is a unified platform where you can host a combination of container images and compute environments for interactive analysis using your preferred tools, like JupyterLab and RStudio Notebooks, Visual Studio Code IDEs, or Xpra remote desktops. Each data studio session is an individual interactive environment that encapsulates the live environment for dynamic data analysis.
+
+On Seqera Cloud, the free tier permits only one running data studio session at a time. To run simultaneous sessions, [contact Seqera][contact] for a Seqera Cloud Pro license.
```

Comment:
The [contact] link isn't working in prod - it just shows up as what it looks like here. Anything we can do to fix it?

Reply:
That's really weird.

Reply:
They render correctly locally, though.

Reply:
It also is working here: https://deploy-preview-258--seqera-docs.netlify.app/platform/24.1/data_studios